### PR TITLE
Update with jump host and readme

### PR DESCRIPTION
**Pull request by jonchenn**
- Add jump host section in readme
- Adding jump host in terraform stage.
- Clean up hardcoded project id and number.

### CODE REVIEW

**Comment by lukmanr** [comment]
LGTM

**Comment by lukmanr** [comment]
I am not sure we can support Python 3.8.  I seem to recall some library dependency on 3.9+.  IMO we should not lower the Python requirement to 3.8.

**Comment by lukmanr** [comment]
Also Python 3.8 is scheduled for EOL in Oct 2024.  3.9 is supported through 2025.

**Comment by jonchenn** [comment]
Updated the Readme to keep with python 3.9 requirement.
The jump host setting can be updated in next PR.

**Comment by jonchenn** [comment]
> Also Python 3.8 is scheduled for EOL in Oct 2024. 3.9 is supported through 2025.

Updated with 3.9 in the jump host.

**Comment by jonchenn** [comment]
Note: the CI/CD is not working as intended. We will fix this in later PR.

File: README.md [hunk 1]
```diff
@@ -67,22 +73,41 @@ gcloud auth application-default login
...
+
+> If you choose to run this setup in your local machine, you can skip this section. However we recommend using a jump host to keep a consistant experience.
 
-Log in to the bastion host.
+Run the following to create a Compute Engine VM as the jump host.
 ```
-# TBD
+sb infra apply 0-jumphost
 ```
+- Please note that this may take 10-15 minutes to install depedencies in the VM.
```
Comment (lavjain) [inline README.md:89]: Latest update: 5-10 minutes

File: README.md [hunk 2]
```diff
@@ -67,22 +73,41 @@ gcloud auth application-default login
...
+- If the file `jumphost-ready` exists, it means the dependencies setup is ready.
+
+Check out the code in the jump host:
+```
+git clone https://github.com/GoogleCloudPlatform/core-solution-services
+```
+
+Update the project ID in the source code:
+```
+sb set project-id $PROJECT_ID
```
Comment (lavjain) [inline README.md:105]: PROJECT_ID may not be set on the jump host
```
export PROJECT_ID=$(gcloud config get project)
echo PROJECT_ID=$PROJECT_ID
sb set project-id $PROJECT_ID
```

File: README.md [hunk 3]
```diff
@@ -67,22 +73,41 @@ gcloud auth application-default login
...
+gcloud compute ssh jump-host --zone=${ZONE} --tunnel-through-iap --project=${PROJECT_ID}
+ls -la /tmp/jumphost-ready
+```
+- If the file `jumphost-ready` exists, it means the dependencies setup is ready.
+
+Check out the code in the jump host:
+```
+git clone https://github.com/GoogleCloudPlatform/core-solution-services
+```
+
```
Comment (lavjain) [inline README.md:102]: We also need `gcloud auth login` and application default login steps on the jump host

**Review by lavjain — APPROVED** [review]

**Pull request merged**